Sources/MongoSwift/ReadPreference.swift

```diff
@@ -96,6 +96,11 @@ public final class ReadPreference {
         self._readPreference = mongoc_read_prefs_new(mode.readMode)
     }
 
+    /// Initialize a new empty `ReadPreference` with the server's default.
```

Comment:
Read preferences are actually a client side feature (we do send them to the server for mongos routing and to determine if secondary reads are okay, though), so there is no "server default". Instead, we can just say "Initializes a new `ReadPreference` with the default mode (primary)".